docs/docs/self-hosting/deploy/docker-compose.yaml

```
@@ -93,6 +124,13 @@ services:
       start_period: 20s
     networks:
       - zitadel
+    ports:
+      - 5432:5432
+    volumes:
+      - 'data:/var/lib/postgresql/data:rw'
 
 networks:
   zitadel:
+
+volumes:
+  data:
```

Add newline to end of file

```suggestion
  data:

```